tests/misc/projects/Issue8366/.gitignore

```diff
@@ -0,0 +1 @@
+cs
```

Comment:
You can generate to `bin`. It's already ignored: 
https://github.com/HaxeFoundation/haxe/blob/0fd449832b2f554a60b488b219b50408136db502/tests/misc/.gitignore#L2

Reply:
Oh, yeah. Found `cs` in the random issue I used as a base last time, never tried to find something better. Thanks, updated!